server/util/log/log.go

```diff
@@ -28,7 +28,7 @@ var (
 	EnableStructuredLogging = flag.Bool("app.enable_structured_logging", false, "If true, log messages will be json-formatted.")
 	IncludeShortFileName    = flag.Bool("app.log_include_short_file_name", false, "If true, log messages will include shortened originating file name.")
 	EnableGCPLoggingFormat  = flag.Bool("app.log_enable_gcp_logging_format", false, "If true, the output structured logs will be compatible with format expected by GCP Logging.")
-	LogErrorStackTraces     = flag.Bool("app.log_error_stack_traces", false, "If true, stack traces will be printed for errors that have them.")
+	LogErrorStackTraces     = flag.Bool("app.log_error_stack_traces", true, "If true, stack traces will be printed for errors that have them.")
```

Comment:
I'm not sure if we want this turned on for everyone since it adds significantly more log output

Reply:
(woops, didn't mean to approve, just meant to comment.)